### PR TITLE
Update and clean-up of references

### PR DESCRIPTION
**Pull request by behnam-zakeri**
In response to #18, this PR includes the following updates:
- Updating the references listed as “In press” or “In review” in the reference list and in-text accordingly.
- Adding some references to global databases for fossil fuel resources and renewable potentials
- Correcting the reference format in text and some typos when needed, including harmonization of Globiom in-text references with MESSAGE

### CODE REVIEW

File: source/overview/temporal.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+MESSAGE models the time horizon 1990 to 2110 in 5- and 10 year time steps where the first 5 periods (1990, 1995, 2000, 2005, 2010) are 5-year periods and the remaining 10 periods are 10-year periods (2020, 2030, 2040, 2050, 2060, 2070, 2080, 2090, 2100, 2110). The first five periods up to 2010 are fully calibrated, i.e. the model has no flexibility to change in these five periods. The 2020 period is partly calibrated so far, some recent trends are included in this time period, but some flexibility remains. In scenario applications the 2010 period is typically fixed to its baseline development so that future climate and energy policy cannot induce changes in the past. The reporting years are the final years of periods which implies that investments that lead to the capacities in the reporting year are the average annual investments over the entire period the reporting year belongs to. In some scenarios, the model has been run with 5-year modeling time steps by the middle of the Century (2015, 2020, 2025, 2030, 2035, 2040, 2045, 2050, 2055).
```
Comment (volker-krey) [inline source/overview/temporal.rst]: I would suggest to remove the years 1990-2005 from the model time horizon, indicating that 2010 is generally used as the base-year. For the sentence starting with "some scenarios", I would suggest to refer instead to a "model version" and here indicate that 2015 is used as the base year.

File: source/overview/temporal.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 MESSAGE can both operate perfect foresight over the entire time horizon, limited foresight (e.g., two or three periods into the future) or myopically, optimizing one period at a time (Keppo and Strubegger, 2010 :cite:`keppo_short_2010`). Most frequently MESSAGE is run with perfect foresight, but for specific applications such as delayed participation in a global climate regime without anticipation (Krey and Riahi, 2009 :cite:`krey_implications_2009`; O'Neill et al., 2010 :cite:`oneill_mitigation_2010`) limited foresight is used.
```
Comment (volker-krey) [inline source/overview/temporal.rst]: Could be useful to add links to the respective parts of the MESSAGEix documentation where foresight is discussed.

File: source/overview/temporal.rst (hunk 1 above)
Comment (behnam2015) [reply source/overview/temporal.rst]: Thanks @volker-krey, I updated the text and removed extyra parts.

File: source/overview/temporal.rst (hunk 2 above)
Comment (behnam2015) [reply source/overview/temporal.rst]: Thanks @volker-krey for this, I added a link to the mathematical formulation.

File: source/energy/conversion/grid.rst [hunk 3]
```diff
@@ -20,7 +20,10 @@ Inter-regional energy transmission infrastructure, such as natural gas pipelines
...
+Seasonal and diurnal load curves and spatial issues such as transmission constraints or renewable resource heterogeneity are treated in a stylized way in the model. 
+The mechanism to represent power system reliability in MESSAGE is based on (Sullivan et al., 2013 :cite:`sullivan_electric_2013`). This method elevates the stylization of temporal resolution by introducing two concepts, 
+peak reserve capacity and general-timescale flexibility (for mathematical representation see this 'Section <https://message.iiasa.ac.at/en/stable/model/MESSAGE/model_core.html#system-reliability-and-flexibility-requirements>`_). To represent capacity reserves in MESSAGE, a requirement is defined that each region build sufficient firm generating capacity to maintain reliability through reasonable load and contingency events. As a proxy for complex system reliability metrics, a reserve margin-based metric was used, setting the capacity requirement at a multiple of average load, based on electric-system parameters. While many of the same issues apply to both electricity from wind and solar energy, the description below focuses on wind.
```
Comment (GamzeUnlu95) [inline source/energy/conversion/grid.rst]: In the web page, the link appears as it is written in the code: 'Section <https://message.iiasa.ac.at/en/stable/model/MESSAGE/model_core.html#system-reliability-and-flexibility-requirements>`_).

Also in the webpage this part appears as it is written (Sullivan et al., 2013 :cite:`sullivan_electric_2013`). And the reference does not appear as a number instead cite command is there.

File: source/energy/resource/renewable.rst [hunk 4]
```diff
@@ -2,11 +2,11 @@
 
 Non-Biomass Renewable Resources
 ================================
-:numref:`tab-depl` shows the assumed total potentials of non-biomass renewable energy deployment (by resource type) in the MESSAGE model. In addition, the assumptions are compared 
-with technical potential estimates from the Global Energy Assessment (Rogner et al., 2012 :cite:`rogner_chapter_2012`). In this context, it is important to note that typical MESSAGE 
+:numref:`tab-depl` shows the assumed total potentials of non-biomass renewable energy deployment (by resource type) in the MESSAGE model. In addition, the technical potential estimates are based on different sources, 
+such as the U.S. National Renewable Energy Laboratory `database <https://www.nrel.gov/grid/data-tools.html>'_ as described in the Global Energy Assessment (Rogner et al., 2012 :cite:`rogner_chapter_2012`).
```
Comment (GamzeUnlu95) [inline source/energy/resource/renewable.rst]: The link for the database can not be clicked and the reference appears with the 'cite' command but not as a number.

Comment (behnam-zakeri) [reply source/energy/resource/renewable.rst]: Thanks for noticing this @GamzeUnlu95. Corrected. Can you please mention which reference is not working?

File: source/energy/conversion/grid.rst (hunk 3 above)
Comment (behnam-zakeri) [reply source/energy/conversion/grid.rst]: Thanks @GamzeUnlu95, the typo is corrected now.

**Pull request merged**